aiida/manage/external/rmq.py

```diff
@@ -145,9 +144,9 @@ def _continue(self, communicator, pid, nowait, tag=None):
 
         try:
             node = load_node(pk=pid)
-        except (exceptions.MultipleObjectsError, exceptions.NotExistent) as exception:
+        except (exceptions.MultipleObjectsError, exceptions.NotExistent):
             LOGGER.exception('Cannot continue process<%d>', pid)
-            raise plumpy.TaskRejected('Cannot continue process: {}'.format(exception))
```

Comment:
Coud you put here, as a comment, a summary of the PR message?

Reply:
Done